src/dhtproto/client/mixins/NeoSupport.d

```diff
@@ -1610,6 +1646,92 @@ template NeoSupport ( )
 
             return res;
         }
+
+        /***********************************************************************
+
+            Struct returned after a RemoveChannel request has finished.
+
+        ***********************************************************************/
+
+        private static struct RemoveChannelResult
```

Comment:
I forgot, but I think `static` here (in non-function context) is redundant.

Reply:
Correct :+1: